alvr/xtask/src/build.rs

```diff
@@ -2,8 +2,37 @@ use crate::{command, dependencies, version};
 use alvr_filesystem::{self as afs, Layout};
 use xshell::{cmd, Shell};
 
+#[derive(Clone, Copy)]
+pub enum Profile {
+    Debug,
+    Release,
+    Distribution,
+}
+
+impl From<(bool, bool)> for Profile {
+    fn from((release, distribution): (bool, bool)) -> Self {
+        if distribution {
+            Self::Distribution
+        } else if release {
+            Self::Release
+        } else {
+            Self::Debug
+        }
+    }
+}
```

Comment:
This can be removed, we don't need a --distribution flag, let's keep it only for package builds.